packages/backend-storage/src/index.ts

```diff
@@ -2,7 +2,7 @@ export * from './factory.js';
 export { StorageResources, AmplifyStorageProps } from './construct.js';
 export {
   RoleAccessBuilder,
-  StorageAccessDefinition,
+  StoragePathAccessDefinition as StorageAccessDefinition,
```

Comment:
Is this to prevent breaking change somewhere else? Do we need to rename export here?

Reply:
Good catch, will update